doc/install.rst

```diff
@@ -290,7 +290,7 @@ Note that those solvers are not enabled by default, please refer to the
 `scikit-learn-intelex <https://intel.github.io/scikit-learn-intelex/what-is-patching.html>`_
 documentation for more details on usage scenarios. Direct export example:
 
-.. prompt:: bash $
+.. prompt:: bash >>>
```

Comment:
```suggestion
.. prompt:: ipythonconsole >>>
```
or maybe `python`. I'd have used `python` but https://sphinx-copybutton.readthedocs.io/en/latest/use.html#automatic-exclusion-of-prompts-from-the-copies makes me think it should be `ipythonconsole`.

Reply:
I initially did not use any `prompt::` at all, and the code was naturally recognized as Python, but it had the undesired side-effect of triggering a CI test that was failing because of the non-availability of the `sklearnex` package. 

I will try a cleaner approach and let you know.

Reply:
`ipythonconsole` is a symbol recognized by the extension sphinx-copybutton.

Here, the `prompt::` is a sphinx-prompt statement. The valid values are specified [here](https://pypi.org/project/sphinx-prompt/#toc-entry-3). Using `ipythonconsole` is not a valid value and would have no specific effect.

I try `python`, and check if the tests pass.

Reply:
Using `prompt:: python >>>` works with all versions of sphinx-prompt. Feel free to reopen this comment as needed.